app/models/namespace.js

```diff
@@ -2,10 +2,28 @@ import { computed, get } from '@ember/object';
 import { inject as service } from '@ember/service';
 import Resource from 'ember-api-store/models/resource';
 import { parseExternalId } from 'ui/utils/parse-externalid';
+import { convertToMillis } from 'shared/utils/util';
+import { parseSi } from 'shared/utils/parse-unit';
 import C from 'ui/utils/constants';
 import { hasMany, reference } from 'ember-api-store/utils/denormalize';
 import StateCounts from 'ui/mixins/state-counts';
 
+export function convertResourceQuota(key, value) {
+  let out;
```

Comment:
I think this function would be more readable (esp if we add more conditions) as a switch statement.